pallets/payment/Cargo.toml

```diff
@@ -21,6 +21,7 @@ sp-std = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v
 frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.13", default-features = false, optional = true }
 orml-traits = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v0.9.13", default-features = false }
 scale-info = { version = "1.0.0", default-features = false, features = ["derive"] }
+virto-primitives = { default-features = false, path = "../../primitives" }
```

Comment:
I think we can have some duplication in the benchmark file and have a similar enum as our runtime asset definition to avoid bringing this dependency, it would defeat the purpose of https://github.com/virto-network/virto-node/issues/154 to bring this back

Reply:
I tried two ways to do it, the duplicate asset definition won't work when we try to generate weights for our runtime since it has to match what we have in runtime. Also tried moving the benchmarks to within the `/runtime` directory. The downside of this is that `cargo test --features runtime-benchmarks` will no longer work. I think we should keep it like this inside our pallet and create a "clean" version of the pallet without benchmarks for orml, or using primitive rust types f(or the purpose of that PR only). I think Acala loosely follows this model since the orml pallets don't have benchmarks, this is left to the teams that add the pallet to the project I believe. Wdyt?

Reply:
Then maybe `optional = true` and add it with the `runtime-benchmarks` feature?

Reply:
I've moved it to `dev-dependencies`, cannot restrict to `runtime-benchmarks` only because we need it for both tests and benchmarks ie we want both `make benchmark` and `cargo test --features runtime-benchmarks` to run

Reply:
As long as both work it's fine. Actually better in `dev-dependencies`, I thought it had to go in the normal dependencies for the benchmarks to work

Reply:
Oops my bad, it doesn't work. Adding as optional dependency + runtime-benchmarks will break the tests. Adding in dev-dependencies will break benchmarks :(